addons/website_theme_install/static/src/less/website_theme_install.less

```diff
@@ -1,42 +1,234 @@
-        padding: 0;
+        padding: 1% 3% 3%;
+        box-shadow: none;
+        background: none;
+        border: none;
+        -webkit-font-smoothing: antialiased;
+
+        .o_browser_bar {
+          background: white;
+          border-bottom: 1px solid darken(@o-theme-kanban-gray, 6%);
+          .size(100%, 20px);
+
+          &:before {
+            content: "•••";
+            display: block;
+            padding-left: 10px;
+            color: darken(@o-theme-kanban-gray, 6%);
+            font: 12px/20px Courier, monospace;
+            letter-spacing: -1px;
+          }
+        }
+
+        &.o_theme_installed .o_theme_preview_top {
+            border-color: @odoo-brand-optional;
+
+            &::after {
+                .o-position-absolute(10px, 10px);
+                .square(20px);
+                display: block;
+                line-height: 20px;
+                text-align: center;
+                background-color: @odoo-brand-optional;
+                border-radius: 100%;
+                font-family: 'FontAwesome';
+                color: white;
+                content: "\f00c";
+            }
+        }
+
+        .o_theme_preview_top {
+            background: white;
+            border: 1px solid darken(@o-theme-kanban-gray, 20%);
+            border-radius: 2px;
+            position: relative;
+            overflow: hidden;
+            will-change: transform;
+            margin-bottom: 5px;
+            .o-transform-origin(center, bottom);
+            .o-transition(all, .1s);
+
+            .o_theme_cover, .o_theme_logo, .o_theme_screenshot {
+                width: 100%;
+                padding-bottom: 127%;
+                background-repeat: no-repeat;
+                background-position: center center;
+                background-color: @gray-lighter;
+                background-size: 40% 32%;
+                .o-transition(opacity, 100ms);
+
+                &.o_theme_cover {
+                    background-size: contain;
+                }
+
+                &.o_theme_screenshot {
+                    background-size: cover;
+                }
+            }
+        }
+
+        .o_theme_preview_bottom {
+            h5, h6 {
+                line-height: 16px;
+            }
+        }
 
-        position: relative;
-        overflow: hidden;
         .o_button_area {
-            .o-position-absolute(@bottom: -100px, @right: 0);
-            .o-transition(bottom, 400ms, ease);
-            width: 100%;
-            padding: @odoo-horizontal-padding/2;
-            background-color: fade(@odoo-brand-secondary, 70%);
+            .o-position-absolute(0, 0);
+            .o-transition(opacity, 100ms);
+            .o-flex-display();
+            .o-flex-flow(row, wrap);
+            .o-justify-content(center);
+            .square(100%);
+            padding: 10% 7%;
             text-align: right;
+            opacity: 0;
+            visibility: hidden;
+
+            .btn {
```

Comment:
@qsm-odoo , I'm removing all rules except that ones related to the `.btn-default` style.